modules/claims_api/spec/lib/claims_api/v2/disability_compensation_validation_spec.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'claims_api/v2/disability_compensation_validation'
+
+# Calling private methods so needed to wrap it in a class
+class TestDisabilityCompensationValidationClass
+  include ClaimsApi::V2::DisabilityCompensationValidation
+end
+
+describe TestDisabilityCompensationValidationClass do
+  subject(:test_526_validation_instance) { described_class.new }
+
+  let(:auto_claim) do
+    JSON.parse(
+      Rails.root.join(
+        'modules',
+        'claims_api',
+        'spec',
+        'fixtures',
+        'v2',
+        'veterans',
+        'disability_compensation',
+        'form_526_json_api.json'
+      ).read
+    )
+  end
+
+  let(:created_at) { Timecop.freeze(Time.zone.now) }
+  let(:form_attributes) { auto_claim.dig('data', 'attributes') || {} }
+
+  describe '#remove_chars' do
+    let(:date_string) { form_attributes['serviceInformation']['servicePeriods'][0]['activeDutyBeginDate'] }
+
+    it 'removes the -DD when the date string has a suffix of -DD' do
+      result = test_526_validation_instance.send(:remove_chars, date_string)
+      expect(result).to eq('2008-11')
+    end
+  end
```

Comment:
The method that was in the file looked like it had been built for an older date format of YYYY-DD-MM and just slipped by since it is rarely used.  Adding this file here to make sure these stay locked in and working as expected.